js_modules/dagit/packages/core/src/launchpad/LaunchpadSetupRoot.tsx

```diff
@@ -51,7 +51,10 @@ const LaunchpadSetupAllowedRoot: React.FC<Props> = (props) => {
       }
       if (queryString.solidSelection instanceof Array) {
```

Comment:
are empty selection arrays okay?

Reply:
i don't think any call site is passing down empty array but i now added a guard just to be safe